From 2f4a07bb849b18cb2ad1d38340769ba4e382e8aa Mon Sep 17 00:00:00 2001
From: John Howard <howardjohn@google.com>
Date: Mon, 14 Mar 2022 12:19:22 -0700
Subject: tf: fix regression in tests run (#37917)

We previously ran all tests. In
https://github.com/istio/istio/commit/3421f15fe499fc4c09d4a476b6fdd80a2c75bfa0,
8 days ago, we accidentally changed this to only `security/` for about 7
  jobs.

This restores those jobs to their former glory, testing all of
`tests/integration/...`.
---
 tests/integration/tests.mk | 11 +----------
 1 file changed, 1 insertion(+), 10 deletions(-)

diff --git a/tests/integration/tests.mk b/tests/integration/tests.mk
index 2a714ee567..02fb974650 100644
--- a/tests/integration/tests.mk
+++ b/tests/integration/tests.mk
@@ -103,17 +103,8 @@ test.integration.kube: test.integration.kube.presubmit
 # Presubmit integration tests targeting Kubernetes environment. Really used for postsubmit on different k8s versions.
 .PHONY: test.integration.kube.presubmit
 test.integration.kube.presubmit: | $(JUNIT_REPORT) check-go-tag
-	$(RUN_TEST) ./tests/integration/security -timeout 30m \
-	${_INTEGRATION_TEST_FLAGS} ${_INTEGRATION_TEST_SELECT_FLAGS} \
-	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
-
-# Defines a target to run a minimal reachability testing basic traffic
-# This test suite should be avoided in favor test.integration.kube.environment
-.PHONY: test.integration.kube.reachability
-test.integration.kube.reachability: | $(JUNIT_REPORT) check-go-tag
-	$(RUN_TEST) ./tests/integration/security/ -timeout 30m \
+	$(RUN_TEST) ./tests/integration/... -timeout 30m \
 	${_INTEGRATION_TEST_FLAGS} ${_INTEGRATION_TEST_SELECT_FLAGS} \
-	--test.run=TestReachability \
 	2>&1 | tee >($(JUNIT_REPORT) > $(JUNIT_OUT))
 
 # Defines a target to run a standard set of tests in various different environments (IPv6, distroless, ARM, etc)
-- 
2.35.3

